consensus/tests/common.rs

```diff
@@ -127,76 +127,83 @@ mod tests {
         assert_eq!(expected, c.last_vrf_hash());
     }
 
-    #[test]
-    #[wasm_bindgen_test]
-    fn selects_longer_chain() {
-        let mut genesis_chain = ProtocolStateChain::default();
-        let mut consensus_state = ConsensusState::default();
-        consensus_state.min_window_density = Length(77);
-        consensus_state.sub_window_densities = vec![
-            Length(1),
-            Length(7),
-            Length(7),
-            Length(7),
-            Length(7),
-            Length(7),
-            Length(7),
-            Length(7),
-            Length(7),
-            Length(7),
-            Length(7),
-        ];
-
-        consensus_state.curr_global_slot = GlobalSlot {
-            slot_number: GlobalSlotNumber(0),
-            slots_per_epoch: Length(7140),
-        };
-
-        let mut prot_state = ProtocolState::default();
-        prot_state.body.consensus_state = consensus_state;
-        genesis_chain.push(prot_state).unwrap();
-
-        let mut chain_at_5001 = ProtocolStateChain::default();
-        let mut consensus_state = ConsensusState::default();
-        consensus_state.min_window_density = Length(43);
-        let densities = vec![
-            Length(5),
-            Length(5),
-            Length(2),
-            Length(5),
-            Length(3),
-            Length(1),
-            Length(5),
-            Length(3),
-            Length(7),
-            Length(6),
-            Length(5),
-        ];
-
-        consensus_state.sub_window_densities = densities.clone();
-
-        consensus_state.curr_global_slot = GlobalSlot {
-            slot_number: GlobalSlotNumber(7042),
-            slots_per_epoch: Length(7140),
-        };
-
-        let mut prot_state = ProtocolState::default();
-        prot_state.body.consensus_state = consensus_state;
-        chain_at_5001.push(prot_state).unwrap();
-
-        let mut chains = vec![];
-        chains.push(chain_at_5001);
-        let canonical = genesis_chain.select_secure_chain(&chains).unwrap();
-        let canonical = canonical.0.get(0).unwrap();
-        assert_eq!(
-            canonical.body.consensus_state.min_window_density,
-            Length(43)
-        );
-        assert_eq!(
-            canonical.body.consensus_state.sub_window_densities,
-            densities
-        );
-    }
+    // @willemolding:
```

Comment:
It would be great to open a discussion around this

Reply:
thanks, will take a look at this in a follow-up PR.